.env.template

```diff
@@ -1,36 +1,36 @@
 #-----[Bridgeipelago Config]-----
 #
 #---[Discord Config]---
-DiscordToken = ''
```

Comment:
Adding whitespace before and after the `=` can cause certain parsers to accidentally treat the whitespace as part of the variable name or the value.

I think in this case with the python parser we're using is smart enough to where it doesn't matter, but typically it's just more conventional to not have the whitespace - so I went ahead and updated `.env.template` to adhere to that.